addons/true-youtube-links/addon.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "True YouTube links",
+  "description": "Replaces links to the embedded site with true YT links",
+  "credits": [
+    {
+      "name": "HTML-Fan/NT"
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["https://scratch.mit.edu/*"],
```

Comment:
This should be /discuss/*

Reply:
Why? There might be a YT-Video linked in the comments or ... well, anywhere. It can happen and in that case it should still work.